styles.css

```diff
@@ -1,3 +1,1317 @@
+    align-items: center;
+    gap: 20px;
+}
+
+.breadcrumbs-container {
+    background-image: url(images/breadcrumbs-img.svg);
+    background-color: #141B22;
+    display: flex;
+    justify-content: center;
+    align-items: center;
+    flex-direction: column;
+    width: 100%;
+    height: 242px;
+    text-align: center;
+    position: relative;
+}
+
+.breadcrumbs {
+    display: flex;
+    justify-content: center;
+    align-items: center;
+    margin-top: 10px;
+}
+
+.breadcrumbs-nav {
+    display: flex;
+    align-items: center;
+}
+
+.breadcrumbs-nav a {
+    text-decoration: none;
+    color: #888B8E;
+    margin: 0 10px;
+}
+
+.about-us-section {
+    box-sizing: border-box;
+    display: inline-block;
+    justify-content: center;
+    margin: auto;
+}
+
+.list-term+.list-description {
+    display: inline-block;
+}
+
+.section-text {
+    margin: auto;
+}
+
+.ot-portrait {
+    border-radius: 50%;
+
+}
+
+.list-item {
+    display: inline-block;
+    width: 250px;
+    height: 150px;
+    margin: 0 5px;
+    background: #141B22;
+    padding: 10px;
+}
+
+.out-team {
+    width: 1410px;
+    height: 824px;
+    display: inline-block;
+    align-items: center;
+
+}
+
+.ot-members {
+    padding: 0;
+}
+
+.ot-member {
+    display: inline-block;
+    justify-content: center;
+    width: 330px;
+    height: 339px;
+    background: #141B22;
+    margin: 7px;
+}
+
+.ot-portrait {
+    border-radius: 50%;
+}
+
+.partners {
+    margin: auto;
+}
+
+.joc-homepage {
+    background-color: #090C11;
+
+}
+
+.join-our-community {
+    border-radius: 8px 0px 0px 0px;
+    margin: auto;
+    background: linear-gradient(268.28deg, #2333DE -1.34%, #0ABE66 100%);
+}
+
+.partner-logo {
+    display: inline-block;
+    margin: 0 5px;
+    padding: 10px;
+}
+.bg-grey {
+    background-color: #141B22;
+}
+
+.social {
+    list-style: none;
+    display: inline-block;
+}
+
+.catalog-menu {
+    background: #141B22;
+}
+
+.colored {
+    color: #21E786;
+    margin: auto;
+    display: inline;
+}
+
+.roadmap-nav {
+    display: flex;
+    justify-content: center;
+    align-items: center;
+}
+
+.phase {
+    display: flex;
+    justify-content: space-between;
+    list-style: none;
+    padding: 0;
+    width: 80%;
+    position: relative;
+}
+
+.phase:before,
+.phase-item {
+    content: '';
+    position: absolute;
+    left: 0;
+    right: 0;
+    height: 2px;
+    width: 100%;
+    background-color: #434343;
+}
+
+.phase-item {
+    position: relative;
+    text-align: center;
+}
+
+.phase-item:before {
+    content: '';
+    position: relative;
+    bottom: 15px;
+    display: block;
+    width: 30px;
+    height: 30px;
+    margin: 0 auto 10px;
+    background-image: url("data:image/svg+xml,%3Csvg width='31' height='30' viewBox='0 0 31 30' fill='none' xmlns='http://www.w3.org/2000/svg'%3E%3Ccircle opacity='0.1' cx='15.5' cy='15' r='15' fill='white'/%3E%3Ccircle cx='15.5' cy='15' r='7.5' fill='%23888B8E'/%3E%3C/svg%3E%0A");
```

Comment:
Instead of using an image here, it’s more efficient to use a gradient.

Using a CSS gradient instead of an image is often more efficient in terms of performance and flexibility. Gradients are scalable, can be adjusted easily in code, and reduce the need for additional image assets, leading to faster load times.